perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -546,4 +547,17 @@ sub bin_services_for_address {
     return \@out;
 }
 
+sub report_allowed {
+    my $dt = shift->clone;
```

Comment:
Small thing but either a comment to explain what's happening here or different names for variables that make it a bit more obvious would be nice

Reply:
Actually I should be using the functions Zurich already have!